### PR TITLE
Add `--local` flag and pass to builds as `isLocalBuild`

### PR DESCRIPTION
**Pull request by tmeasday**
I'm not sure if we want the flag (it could just be a node API option), but I thought it could be helpful for debugging. I didn't add it as an input to the action. Not sure if that's confusing.

Depends on https://github.com/chromaui/chromatic/pull/7576

### CODE REVIEW

File: node-src/lib/parseArgs.ts [hunk 1]
```diff
@@ -74,6 +75,7 @@ export default function parseArgs(argv: string[]) {
         exitOnceUploaded: { type: 'string' },
         exitZeroOnChanges: { type: 'string' },
         externals: { type: 'string', isMultiple: true },
+        local: { type: 'boolean' },
         ignoreLastBuildOnBranch: { type: 'string' },
```
Comment (ghengeveld) [inline node-src/lib/parseArgs.ts:79]:
```suggestion
        ignoreLastBuildOnBranch: { type: 'string' },
        local: { type: 'boolean' },
```
Alphabetical order.

File: node-src/lib/parseArgs.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ export default function parseArgs(argv: string[]) {
       --exit-zero-on-changes [branch]           If all snapshots render but there are visual changes, exit with code 0 rather than the usual exit code 1. Only for [branch], if specified. Globs are supported via picomatch.
       --externals <filepath>                    Disable TurboSnap when any of these files have changed since the baseline build. Globs are supported via picomatch. This flag can be specified multiple times. Requires --only-changed.
       --ignore-last-build-on-branch <branch>    Do not use the last build on this branch as a baseline if it is no longer in history (i.e. branch was rebased). Globs are supported via picomatch.
+      --local                                   Create a "local" build, that will not affect other user's baselines.
```
Comment (ghengeveld) [inline node-src/lib/parseArgs.ts:30]:
```suggestion
      --local                                   Create a "local" build, which will not affect other user's baselines.
```

File: node-src/tasks/initialize.ts [hunk 3]
```diff
@@ -65,6 +65,7 @@ export const announceBuild = async (ctx: Context) => {
         ...commitInfo,
         committedAt: new Date(committedAt),
         ciVariables: ctx.environment,
+        isLocalBuild,
```
Comment (ghengeveld) [inline node-src/tasks/initialize.ts:68]:
```suggestion
        isLocalBuild: local,
```

File: node-src/types.ts [hunk 4]
```diff
@@ -69,6 +70,7 @@ export interface Options {
   autoAcceptChanges: true | string;
   exitZeroOnChanges: true | string;
   exitOnceUploaded: true | string;
+  isLocalBuild: boolean;
```
Comment (ghengeveld) [inline node-src/types.ts:73]:
```suggestion
  local: boolean;
```

**Review by ghengeveld — APPROVED** [review]
Some minor nitpicks but otherwise LGTM.

**Comment by tmeasday** [comment]
@ghengeveld as discussed on Slack we dropped the `--local` option and now the only way to get a local build is to use the node API. We can revisit this later if we like.

**Pull request merged**